tests/smoke-tests/pom.xml

```diff
@@ -28,7 +28,7 @@
 
    <properties>
       <activemq.basedir>${project.basedir}/../../</activemq.basedir>
-      <sts-surefire-extra-args />
+      <sts-surefire-extra-args>-Ddistribution.lib="${activemq.basedir}/artemis-distribution/target/apache-artemis-${project.version}-bin/apache-artemis-${project.version}/lib"</sts-surefire-extra-args>
```

Comment:
This isnt the right place to declare this, as its used as a profile property to add extra args on JVMs that need them [1], so putting it here actually means those JVMs wont get this config. Add a specific property for this and reference it in the surefire config at: https://github.com/apache/activemq-artemis/blob/main/tests/smoke-tests/pom.xml#L1335

[1] https://github.com/apache/activemq-artemis/blob/main/tests/smoke-tests/pom.xml#L1348